### PR TITLE
(#11) Add icons to the ui-kit package

### PR DESCRIPTION
**Pull request by KonradSzwarc**

### CODE REVIEW

File: development/storybook/src/components/IconCard.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export type IconCardProps = {
+  icon: ComponentType;
+};
+
+export const IconCard = ({ icon: Icon }: IconCardProps) => {
+  return (
+    <Box my={24} fontSize={24} fontFamily="sans-serif">
+      <Flex align="center">
+        <Icon />
```
Comment (mecies) [inline development/storybook/src/components/IconCard.tsx:12]: Do I understand correctly, is this a template for auto-generating icon components and `Icon` is placeholder?

File: packages/ui-kit/package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
     "lint:check": "eslint \"src/**/*.{js,ts,tsx}\"",
     "lint:fix": "eslint \"src/**/*.{js,ts,tsx}\" --fix",
     "typecheck": "tsc --noEmit",
-    "check-updates": "ncu"
+    "check-updates": "ncu",
```
Comment (mecies) [inline packages/ui-kit/package.json:13]: Didn't know that lib 👁️ 👁️

File: packages/ui-kit/src/icons/components/Check.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React, { forwardRef } from 'react';
+
+export const CheckIcon = forwardRef((props: React.SVGProps<SVGSVGElement>, svgRef?: React.Ref<SVGSVGElement>) => (
```
Comment (mecies) [inline packages/ui-kit/src/icons/components/Check.tsx:3]: This `SVGSVGElement` name in all components is weird - do we have to repeat the `SVG` twice? 🤔

**Review by mecies — CHANGES_REQUESTED** [review]
Good job with the icons 🥇 , left some questions and 1 potential typo, please address and will approve 😄  🎬

File: packages/ui-kit/src/icons/components/Check.tsx (hunk 3 above)
Comment (mecies) [reply packages/ui-kit/src/icons/components/Check.tsx:3]: Wow this is weird
https://developer.mozilla.org/en-US/docs/Web/API/SVGSVGElement

**Review by celinawierzbicka — APPROVED** [review]

**Review by kacperjanusz — APPROVED** [review]

**Review by mecies — APPROVED** [review]
After checking mdn docs I approved 😄

File: packages/ui-kit/src/icons/components/Check.tsx (hunk 3 above)
Comment (KonradSzwarc) [reply packages/ui-kit/src/icons/components/Check.tsx:3]: Yea, that's weird. It's something that the library generates, so I trust it 😅

File: development/storybook/src/components/IconCard.tsx (hunk 1 above)
Comment (KonradSzwarc) [reply development/storybook/src/components/IconCard.tsx:12]: As you can see in `development/storybook/src/stories/Icons.stories.mdx` icon is just a react component passed by props

**Pull request merged**